Components/Alert/alert.html

```diff
@@ -0,0 +1,111 @@
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Alert</title>
+    <link rel="stylesheet" href="/index.css">
+    <link rel="stylesheet" href="/css/sidebar.css">
+    <link rel="stylesheet" href="/css/components.css">
+</head>
+
+<body>
+    <header class="header">
+        <div class="logo">
+            <h1>Handy UI</h1>
+        </div>
+        <div class="main-links">
+            <a href="https://github.com/kotesh-arya"><img class="header-icon"
+                    src="https://img.icons8.com/windows/64/ffffff/github.png" /></a>
+            <a href="https://twitter.com/Kotesh_Mudila"><img class="header-icon"
+                    src="https://img.icons8.com/ios-glyphs/60/ffffff/twitter--v1.png" /></a>
+            <a href="https://www.linkedin.com/in/koteswara-rao-mudila-7aa740159/"><img class="header-icon"
+                    src="https://img.icons8.com/ios-glyphs/60/ffffff/linkedin.png" /></a>
+        </div>
+    </header>
+    <main>
+        <div class="side-bar">
+            <ul>
+                <li>
+                    <h3>Get-started</h3>
+                </li>
+                <hr>
+                <li><a class="sidebar-item" href="/Get-Started/introduction.html">Introduction</a></li>
+                <li><a class="sidebar-item " href="/Get-Started/typography.html">Typography</a></li>
+                <li><a class="sidebar-item" href="">Colours</a></li>
+                <li>
+                    <h3>Components</h3>
+                </li>
+                <hr>
+                <li><a class="sidebar-item" href="/Components/Avatar/avatar.html">Avatar</a></li>
+                <li><a class="sidebar-item link-active" href="/Components/Alert/alert.html">Alert</a></li>
+                <li><a class="sidebar-item" href="">Badge</a></li>
+                <li><a class="sidebar-item" href="">Button</a></li>
+                <li><a class="sidebar-item" href="">Card</a></li>
+                <li><a class="sidebar-item" href="">Image</a></li>
+                <li"><a class="sidebar-item" href="">Input</a></li>
+            </ul>
+        </div>
+        <section class="avatar-section">
+            <div class="section-title">
+                Alert
+            </div>
+            <div class="section-subtitle">
+                An alert displays a short, important message in a way that attracts the user's attention without
+                interrupting the user's task.
+            </div>
+            <hr>
+            <article>
+                <div class="article-title">
+                    Simple Alert
+                </div>
+                <div class="article-subtitle">
+                    Simple Alerts can be created using <span>alert</span> class, followed by the variant name.
+                </div>
+                <div class="example-div">
+                    <div class="alert primary">Primary-This is a Primary Alert!</div>
```

Comment:
suggestion: you can use descriptive class names like "alert-primary" instead of just "primary" to make code more readable.

Reply:
I will add them 👍🏻